Linux/commands/README.md

```diff
@@ -57,6 +57,13 @@
 
 ### System commands
 - `ps aux` - processes which are running
+- `systemctl [option] [service]` - interact with a process
+    - We can do 4 `option` with `systemctl`
+        - start
+        - stop
+        - enable
+        - disable
```

Comment:
Earlier by mistake, I had added it as 'disabel' but after raising the PR, I have again fixed it to 'disable'. Thank you.